tests/microbenchmarks.py

```diff
@@ -244,6 +255,7 @@ class Z: pass
         g = 3.0
     """), timed=True)
 
+
 def test_method(test_case):
     print("Running, test_method")
     test_case.runAsJava(adjust("""
```

Comment:
At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils